### DISCUSSION TITLE
[Feature] Undo "add to album"/"delete"

### DISCUSSION THREAD

**Discussion by enboig** [immich-app/immich#2685] [Feature Request]
### The feature

When adding an image to an album, it would be nice having a 5 second bubble to undo it if you clicked on the wrong album.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by AZweimiller** [comment]
I've just started using immich full-time and this was one of the first things I noticed. Feels especially needed since there is no "are you sure?" dialog when deleting photos.

**Comment by NicholasFlamy** [comment]
I support this because it would be a more efficient user experience than a confirmation prompt. Hitting `Delete` followed by `Yes, I'm Sure` is slower than hitting `Delete` but if you changed your mind or made a mistake `Undo` in the popup about the asset trashed, added to album, or archived would save you.

**Reply by NicholasFlamy** [reply]
Nevermind this is mobile we're talking about

Ignore the below:
Forgot to mention that there is a popup to bring you to the album, just no popup to undo delete.
